packages/entities/entities-plugins/src/definitions/metadata.ts

```diff
@@ -748,6 +748,14 @@ export const PLUGIN_METADATA: Record<string, Omit<PluginMetaData<I18nMessageSour
     scope: [PluginScope.GLOBAL, PluginScope.SERVICE, PluginScope.ROUTE, PluginScope.CONSUMER, PluginScope.CONSUMER_GROUP],
     imageName: 'oauth2',
   },
+  'confluent': {
+    descriptionKey: 'plugins.meta.confluent.description',
+    group: PluginGroup.TRANSFORMATIONS,
+    isEnterprise: true,
+    nameKey: 'plugins.meta.confluent.name',
+    scope: [PluginScope.GLOBAL, PluginScope.SERVICE, PluginScope.ROUTE, PluginScope.CONSUMER],
+    imageName: 'confluent',
```

Comment:
nit: if the plugin name and the icon's file name are the same, `imageName` can be removed.

Reply:
Nice catch. I've updated the pull request